### DISCUSSION TITLE
Can't install with operatorhub ( v1.36.0 )

### DISCUSSION THREAD

**Discussion by ryudwig** [kiali/kiali#4099] [General]
hello, 
I am using kiali operator for maistra service mesh. (OKD cluster)
last Friday, I tried to set up new OKD cluster ( include kiali operator )
but  I could not get a kiali operator. 
The problem seems to be that v1.36.0 registered in operatorhub is not registered in quay.io.
( So, the installation was successful through the helm chart (v1.35), but it did not work normally with the maistra service mesh. )
I would be very grateful if you could solve the problem quickly.

![image](https://user-images.githubusercontent.com/52852208/121828527-d1485180-ccfa-11eb-9c5e-b1f5668e62a2.png)
![image](https://user-images.githubusercontent.com/52852208/121828536-d73e3280-ccfa-11eb-9ef0-233b9eb9b245.png)
![image](https://user-images.githubusercontent.com/52852208/121828547-defdd700-ccfa-11eb-8550-5c9121365bbe.png)

**Comment by jmazzitelli** [comment]
Already reported.. please see: https://github.com/kiali/kiali/issues/4089 -- workaround reported in that issue. This will be fixed this week.

**Reply by ryudwig** [reply]
It hasn't been resolved yet. You said that kiali operator has few users, I am eagerly waiting for this problem to be resolved

**Reply by jmazzitelli** [reply]
I'm looking into what happened to our 1.36.0 build that should have been released on Friday. This should be resolved soon.

**Reply by jmazzitelli** [reply]
Looks like we had a failure in the backend infrastructure responsible for creating the build. We believe we fixed that situation - a 1.36.0 build is being performed. Should be available soon, barring any additional issues that crop up.

**Reply by jmazzitelli** [reply]
1.36.0 is available on quay.io now.
